src/core/libmaven/mzMassSlicer.cpp

```diff
@@ -109,99 +115,184 @@ void MassSlices::algorithmB( MassCutoff *massCutoff,int rtStep ) {
         }
 
         string num;
-        if(i==0) num = "st";
-        else if(i==1) num = "nd";
-        else if(i==2) num = "rd";
-        else num = "th ";
+        if (i==0) {
+            num = "st";
+        } else if (i==1) {
+            num = "nd";
+        } else if (i==2) {
+            num = "rd";
+        } else {
+            num = "th";
```

Comment:
i = 20 will be 21th :P

Sounds like just writing "slice number i+1" will be easier. No need to correct it, just nitpicking here.

Reply:
Ya that sounds better. I thought about doing some modulus math to get the right suffix but then 11 to 19 would need special treatment. Instead I will change the message to just display numbers.